IPython/core/display.py

```diff
@@ -731,6 +730,9 @@ def __init__(self, data=None, url=None, filename=None, format=None,
                 if data[:2] == _JPEG:
                     format = self._FMT_JPEG
 
+        if not format:
+            format=self._FMT_JPEG
```

Comment:
This doesn't feel right. If we can't infer the format, I think we should throw an error rather than assuming it's JPEG. "In the face of ambiguity, refuse the temptation to guess" and all that. ;-)